package.json

```diff
@@ -78,7 +82,7 @@
     "@babel/preset-typescript": "7.9.0",
     "@types/rc-tree": "1.11.3",
     "@types/react-router": "5.1.5",
-    "@types/react-router-dom": "5.1.4",
+    "@types/react-router-dom": "^5.1.5",
```

Comment:
should we pin these deps?

Reply:
Oops, yeah. I just added a bunch of type definitions without looking too closely at what was here already.